lib/active_model/serializer.rb

```diff
@@ -54,10 +57,11 @@ def self.type(type)
     end
 
     def self.attributes(*attrs)
+      options = attrs.last.class == Hash ? attrs.pop : {}
       attrs = attrs.first if attrs.first.class == Array
 
       attrs.each do |attr|
-        attribute(attr)
+        attribute(attr, options)
```

Comment:
I would usually agree, but since we already support variable number of arguments *and* array of symbols as parameters to `attributes`, I don't believe the added cost is that high.